locales/zh.yml

```diff
@@ -56,6 +56,8 @@ toolCard:
   new: '新'
 search:
   label: '搜索'
+transition:
+  placeholder: '请输入工具名或者命令……'
```

Comment:
Hi, I'm a new user of it-tools and I had the idea to add more translations of it before I found your excellent work.
I reviewed your change. It's a wonderful idea to move the placeholder of the popup search input box into the locale file. But also, I do realize that this transaction is newly added, and doesn't exist in other language `yml` files under the locales folder. 
Please also add this key in other language files, so this PR won't break the function for other languages.